test/helpers/url-helpers.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as url_helpers from "../../src/helpers/url-helpers"
+
+describe("url_helpers.isValidExternalURL", () => {
+    test("0", () => {
```

Comment:
instead of `"0"`, could you please use this string to explain what the test scenario is?